src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_core.clj

```diff
@@ -102,11 +102,13 @@
             (when-not (empty? events)
               (log/info (trs "Got {0} event(s) for watched-path {1}"
                              (count orig-events) (.watchable watch-key)))
-              (log/debug (trs "Events:\n{0}"
-                              (pprint-events events)))
-              (log/trace (trs "orig-events:\n{0}"
-                              (ks/pprint-to-string
-                                (map clojurize-for-logging orig-events))))
+              (log/debug (format "%s\n%s"
```

Comment:
any reason not to just use `log/debugf` here and do away with the format call? similarly with trace vs tracef below.